libraries/adaptive-expressions/src/parser/expressionParser.ts

```diff
@@ -130,6 +129,27 @@ export class ExpressionParser implements ExpressionParserInterface {
             return new Constant(this.evalEscape(text));
         }
 
+        public visitJsonCreationExp(context: ep.JsonCreationExpContext): Expression {
+            let expr: Expression = this.makeExpression(ExpressionType.Json, new Constant('{}'));
+            if (context.keyValuePairList()) {
+                for (const kvPair of context.keyValuePairList().keyValuePair()) {
+                    let key = '';
+                    const node = kvPair.key().children[0];
+                    if (node instanceof TerminalNode) {
+                        if (node.symbol.type === ep.ExpressionAntlrParser.IDENTIFIER) {
+                            key = node.text;
+                        } else {
+                            key = node.text.substring(1, node.text.length - 1);
+                        }
+                    } 
+                    
+                    expr = this.makeExpression(ExpressionType.SetProperty, expr, new Constant(key), this.visit(kvPair.expression()));
```

Comment:
we should consider if there exists the same keys and what should happen.
Throw exception, or just simply use `SetProperty` to override the previous one.